models/sessions/scratch/snowflake/snowplow_web_sessions_this_run.sql

```diff
@@ -56,8 +56,9 @@ with session_firsts as (
         mkt_campaign as mkt_campaign,
         mkt_clickid as mkt_clickid,
         mkt_network as mkt_network,
+        regexp_substr(page_url, 'utm_source_platform=([^?&#]*)', 1, 1, 'e') as mkt_source_platform,
+        {{ channel_group_query() }} as default_channel_group,
```

Comment:
@misterpig my regex is iffy at best, but this seems to imply we should be using the 'e' modifier, https://stackoverflow.com/questions/16986331/can-someone-explain-the-e-regex-modifier ?

Reply:
we should be using the `/e` modifier in the regex?
would you be referencing the e parameter in the regex function?